File: Sesion 7/Marco teorico/8herencia_public/8.py
class Vehiculo:
    def conducir(self):
        print("¡El vehículo está en movimiento!")

class Automovil(Vehiculo):
    def encenderLuces(self):
        print("Luces encendidas")

mi_auto = Automovil()
mi_auto.conducir()  
mi_auto.encenderLuces()  

File: Sesion 7/Marco teorico/8herencia_public/8.java
import java.util.*;

class Vehiculo {
    public void conducir() {
        System.out.println("¡El vehículo está en movimiento!");
    }
}

class Automovil extends Vehiculo {
    public void encenderLuces() {
        System.out.println("Luces encendidas");
    }
}

public class Main {
    public static void main(String[] args) {
        Automovil miAuto = new Automovil();
        miAuto.conducir(); 
        miAuto.encenderLuces(); 
    }
}
